.travis/build.test.sh

```diff
@@ -28,8 +31,11 @@ DEBIAN_FRONTEND=noninteractive apt-get -y -qq install \
 
 curl -sOL https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-installer-${BAZEL_OS}-x86_64.sh
 chmod +x bazel-${BAZEL_VERSION}-installer-${BAZEL_OS}-x86_64.sh
-./bazel-${BAZEL_VERSION}-installer-${BAZEL_OS}-x86_64.sh
+
+# Install bazel, display log only if error
+./bazel-${BAZEL_VERSION}-installer-${BAZEL_OS}-x86_64.sh 2>&1 > bazel_install.log || (cat bazel_install.log && false)
```

Comment:
The bazel install log is pretty long and I don't think we need it if install went fine, this should only print the log if there was an error. Does that seem ok?